### ISSUE TITLE
Make optional? 

### ISSUE THREAD

**Issue by lorensr** [hharnisc/hypercwd#1]
Hi Harrison, nice extension 😊 Looks like it always happens? Would be nice if it were optional, like only happening on cmd-alt-T, or some user-defined shortcut.

**Comment by hharnisc** [comment]
Hey @lorensr! Cool to see some familiar faces from the Meteor community using HyperTerm! 😄 

I went this direction since this is the default on Terminal.app. Is the cmd-alt-T key something from another terminal app like ITerm? Would love to get an idea of what that most common patterns are to build out a good default and make it easy to customize.

On that note, I think it makes sense to make the keyboard shortcut configurable. Thank you for the input!

**Comment by lorensr** [comment]
Nope, made it up! Oh, didn't remember that Terminal default – I usually
have a cd at the end of my .bash_profile 😊

**Comment by jimthedev** [comment]
iTerm defaults to your home directory but lets your configure it on a profile by profile basis http://nateeagle.com/2013/03/08/open-new-tabs-in-iterm-in-the-current-directory/

Since iTerm lets you bind a keyboard shortcut to open a new tab using a specific profile. So, if you want to have different initial directories for new tabs using different keyboard shortcuts then you get this for free. 

It would be nice to have an option to configure the keyboard shortcuts for hyperterm's new window functionality.

**Comment by hharnisc** [comment]
@jimthedev I'm picturing a mapping between keyboard shortcuts and directories

```js
{
    "shortcuts": {
        "cmd-t": null,  // something like null would be previous session's directory
        "cmd-alt-t": "~/",
        "cmd-ctrl-t": "~/Documents"
        /* other shortcuts */
   }
}
```

Using something like [mousetrap](https://craig.is/killing/mice) to hook up the shortcuts to create new sessions with the directory it's mapped to.

Seems like that might also handle @lorensr's idea from feedback as well.
